python/equistore-operations/equistore/operations/join.py

```diff
@@ -17,9 +18,42 @@ def join(tensors: List[TensorMap], axis: str):
     `properties` dimension and for ``axis='samples'`` they will be the along the
```

Comment:
```
`axis='properties'`` it will be the `tensor_maps` will be joined along the
```
->
```
`axis='properties'`` the tensor maps in `tensors` will be joined along the
```